__tests__/app.test.js

```diff
@@ -499,4 +499,24 @@ describe("GET /api/articles?topic", () => {
         expect(body.msg).toBe('Resource not found');
       });
   });
-})
\ No newline at end of file
+})
+describe("GET /api/articles/:article_id comment_count", () => {
+  test("status:200, responds with the total count of all the comments from the specified article id", () => {
+    return request(app)
+    .get("/api/articles/1")
+      .expect(200)
+      .then(({body}) => {
+        const {article} = body
+        expect(article).toMatchObject({
+          article_id: 1,  
+          author: expect.any(String),
+          title: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+          comment_count: 11
+          });
```

Comment:
I cannot understate how happy I am that this is a separate test and not an amend in an old one.